workers/unity/Assets/Playground/Scripts/Bootstrap.cs

```diff
@@ -28,6 +29,7 @@ public void Awake()
             SetupInjectionHooks(); // Register hybrid injection hooks
             PlayerLoopManager.RegisterDomainUnload(DomainUnloadShutdown, 10000); // Clean up worlds and player loop
 
+            CompilationPipeline.assemblyCompilationStarted += CompilationStarted;
```

Comment:
Should this also be removed from `assemblyCompilationStarted`?

Reply:
Added removal as well :)

Reply:
This is unity editor only, and will fail compilation for standalone!

Reply:
Please move this into the `UNITY_EDITOR` block

Reply:
Ahh thanks for that, moved.